examples/esm/index.mjs

```diff
@@ -66,10 +68,14 @@ import Arweave from 'arweave';
   });
 
   const jwk = await arweave.wallets.generate();
-
-  const processId = await spawnANT({
-    signer: new ArweaveSigner(jwk),
-  });
+  let processId;
+  try {
+    processId = await spawnANT({
+      signer: AOProcess.createAoSigner(new ArweaveSigner(jwk)),
```

Comment:
i'm in favor of moving `createAoSigner` to a standalone utility function that we export and use in the class. there's no need for it to be static anymore

Reply:
https://github.com/ar-io/ar-io-sdk/pull/175/commits/8e95eddf5723b0fdbf870688556f76a3e80bb6fa